rules/section-order.js

```diff
@@ -0,0 +1,70 @@
+var rule = require('unified-lint-rule')
+// var position = require('unist-util-position')
+var stringify = require('unist-util-stringify-position')
+var schema = require('../util/schema')
+
+module.exports = rule('standard-readme:section-order', sectionOrder)
+
+var order = [
+  'table-of-contents',
+  'security',
+  'background',
+  'install',
+  'usage',
+  'api',
+  /* Note: maintainer and maintainers don’t matter in this case... */
+  'maintainer',
+  'maintainers',
```

Comment:
Should be optional?

Reply:
yes, only the ones used in the document are checked by this rule.

Reply:
This list contains all the headings that can be used: [only the used ones](https://github.com/RichardLitt/standard-readme-preset/blob/358db2b0cc81f6b328dfdea0dd2659d23f050c98/rules/section-order.js#L39) are checked later.